apps/bluetooth/js/modules/pair_manager.js

```diff
@@ -216,6 +215,57 @@ define(function(require) {
     },
 
     /**
+     * Watch 'onpairingaborted' dom event for pairing aborted.
+     * A handler to trigger when pairing fails due to one of 
+     * following conditions:
+     * - authentication fails
+     * - remote device down (bluetooth ACL becomes disconnected)
+     * - internal error happens
+     *
+     * @access private
+     * @memberOf PairManager
+     */
+    _watchOnpairingaborted: function() {
+      if (!this._defaultAdapter) {
+        return;
+      }
+      
+      this._defaultAdapter.onpairingaborted = this._onPairingAborted.bind(this);
```

Comment:
I would suggest to use `addEventListener` so it won't be overridden  by mistake.